db/institutions.yml

```diff
@@ -149,3 +149,11 @@ ucopenhagen:
   name: University of Copenhagen
   state: ''
   zip: ''
+uwashington:
+  aka: [University of Washington]
```

Comment:
Isn't UW known by any other names?  The Bulldogs or something?

Reply:
It's the UW Huskies :dog2:

Reply:
The Huskies are a mascot, not an AKA, so I am opting not to add that.

Reply:
Perhaps UW, though?

Reply:
OK, no problem.  But what about UW?